src/pages/ClusterPage/explore.js

```diff
@@ -121,15 +124,20 @@ export default class ExploreCluster extends Component {
 		const arcURL = this.state.arc.url
 			? this.state.arc.url.slice(0, -1)
 			: '';
-		let mainURL = 'https://arc-dashboard.appbase.io';
+		let mainURL = this.state.redirectToBilling
+			? 'https://arc-dashboard.appbase.io/cluster/billing'
+			: 'https://arc-dashboard.appbase.io';
+
 		if (urlParams && urlParams.view) {
 			const nestedRoute = urlParams.view.startsWith('/')
 				? urlParams.view.replace('/', '')
 				: urlParams.view;
 
 			mainURL = `${mainURL}/${nestedRoute}`;
 		}
-		const url = `${mainURL}/?url=${arcURL}&username=${this.state.arc.username}&password=${this.state.arc.password}&cluster=${this.state.cluster}&header=false&showHelpChat=false`;
+		const url = this.state.redirectToBilling
```

Comment:
Why we are checking this ? We can just override the `mainURL` value

Reply:
You see the difference is there when its root of arc dashboard where its need to be redirected, then we need to add `/` but if its cluster/billing where its needed to be redirected the we don't need `/` as there is `cluster/billing` for arc dashboard.

The only difference
```js
		${mainURL}? ...
```
```js
       ${mainURL}/? ...
````